src/js/common/schemaform/helpers.js

```diff
@@ -318,7 +318,7 @@ export function updateRequiredFields(schema, uiSchema, formData) {
     const newRequired = Object.keys(schema.properties).reduce((requiredArray, nextProp) => {
       const field = uiSchema[nextProp];
       if (field && field['ui:required']) {
-        const isRequired = field['ui:required'](formData);
+        const isRequired = field['ui:required'](pageData, formData);
```

Comment:
When passing the data to the callback, should we separate them out by `pageData` and `formData`? Can we just send `formData`? Currently, this `formData` isn't broken up by page, so it can be accessed the same way as `pageData`.

Theoretically, this shouldn't cause an issue since all the property names should be unique through all the pages, so when we flatten all page data into a single object, we shouldn't get any overrides, but I went with providing two variables for now in case there was a pressing need to make sure the data we're looking at was really in **this** page.

Perhaps if we keep it two variables, we switch the order?